lib/Dialect/Seq/SeqOps.cpp

```diff
@@ -1004,6 +1019,63 @@ FirMemory::FirMemory(hw::HWModuleGeneratedOp op) {
   initIsInline = op->getAttrOfType<BoolAttr>("initIsInline").getValue();
 }
 
+LogicalResult InitialOp::verify() {
+  auto *terminator = this->getBody().front().getTerminator();
```

Comment:
Do we need to verify, this is a `YieldOp` ?

Reply:
It's verified/ensured by `SingleBlockImplicitTerminator<YieldOp>`